README.md

```diff
@@ -5,17 +5,44 @@
 
 **NOTE:** For the latest stable [README.md](https://github.com/rapidsai/cucim/blob/main/README.md) ensure you are on the `main` branch.
 
+- [GTC 2021 cuCIM: A GPU Image I/O and Processing Toolkit [S32194]](https://www.nvidia.com/en-us/gtc/catalog/?search=cuCIM#/)
+  - [video](https://gtc21.event.nvidia.com/media/cuCIM%3A%20A%20GPU%20Image%20I_O%20and%20Processing%20Toolkit%20%5BS32194%5D/1_fwfxd0iu)
+
 ## Install cuCIM
 
 ### Conda
 
 #### Conda (stable)
 
-> conda create -n cucim -c rapidsai -c conda-forge/label/cupy_rc cucim
+> conda create -n cucim -c rapidsai -c conda-forge/label/cupy_rc -c conda-forge cucim cudatoolkit=11.0
```

Comment:
Would it make sense to template the CUDA version and perhaps specify below that the CUDA version needs to be `11.0+`?

```suggestion
> conda create -n cucim -c rapidsai -c conda-forge/label/cupy_rc -c conda-forge cucim cudatoolkit=<CUDA version>
```

Reply:
yes, it makes sense. I didn't test with cuda 11.1+ but I guess it may work. (but for GDS enablement, I may need to check it in the future.)